src/style.css

```diff
@@ -0,0 +1,129 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc) format("truetype");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf) format("truetype");
+  font-style: normal;
+  font-weight: 600;
+
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+ 
+}
+
+.search-form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+
+}
+
+.search-form_big {
+  width: 100%;
+  height: 70px;
+
+}
+
+.search-form_small {
+  width: 100%;
+  height: 42px;
+
+}
+
+.search-form_label {
+  position: relative;
+  display: block;
+  width: 100%;
+
+}
+
+.search-form_label:after {
+  position: absolute;
+  content: "";
+  display: block;
+}
+
+.search-form_label_big:after {
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
```

Comment:
Check your properties order, please https://9elements.com/css-rule-order/

Reply:
didn't fix it